pkg/services/object/head/remote.go

```diff
@@ -71,7 +71,7 @@ func (h *RemoteHeader) Head(ctx context.Context, prm *RemoteHeadPrm) (*object.Ob
 		return nil, err
 	}
 
-	c, err := h.clientCache.Get(addr)
+	c, err := h.clientCache.Get(prm.node)
```

Comment:
Maybe we need to reconsider do we need `addr` there at all? I see in this function

```go
	addr, err := prm.node.HostAddrString()
	if err != nil {
		return nil, err
	}
        ...
	p := new(client.ObjectHeaderParams).
		WithAddress(prm.commonHeadPrm.addr).
```

Looks a bit weird why other components need host address.

Reply:
hm, ok, it is only used in building error message, so deleted retrieving `HostAddr` string and start using `prm.node` instead